src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java

```diff
@@ -21,6 +23,32 @@ public void setUp() throws Exception {
         }
     }
 
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+
+    @Test
+    public void testImportEntriesException() throws IOException {
+        thrown.expect(IOException.class);
+
+        CopacImporter importer = new CopacImporter();
+
+        @SuppressWarnings("unused")
+        List<BibtexEntry> entries = importer.importEntries(null, new OutputPrinterToNull());
+    }
+
+    @Test
+    public void testGetFormatName(){
+        CopacImporter importer = new CopacImporter();
+        Assert.assertEquals("Copac",importer.getFormatName());
+    }
+
+    @Test
+    public void testGetCLIId(){
+        CopacImporter importer = new CopacImporter();
+        Assert.assertEquals("cpc",importer.getCLIId());
```

Comment:
Please format the file according to the Eclipse rules. The effect would be, among other things, that this line is formatted the following way: `Assert.assertEquals("cpc", importer.getCLIId());`